tests/unit/query_graph/sql/adapter/base_adapter_test.py

```diff
@@ -89,6 +89,7 @@ def test_group_by(self):
                     parent_expr=(get_qualified_column_identifier("parent", "TABLE")),
                     parent_dtype=DBVarType.ARRAY,
                     result_name="result",
+                    parent_cols=[(get_qualified_column_identifier("parent", "TABLE"))],
```

Comment:
a lot of these look similar with the `parent_expr` because of how the test is setup. in reality, the `parent_expr` for aggregations would include the aggregation function, eg. `sum(col)`, and the parent_cols will just be `col`.